test/load/packer/README.md

```diff
@@ -0,0 +1,22 @@
+## Quick start
+
+# Consul AMI:
+
+Within the `consul-ami/` directory
+
+1) Update the `variables` section of the `consul.json` Packer template to configure the AWS region and datadog api key you would like to use. Feel free to reference this article to find your [datadog API key](https://docs.datadoghq.com/account_management/api-app-keys/#api-keys). 
+2) Run `packer build consul.json`. 
+
+When the build finishes, it will output the IDs of the new AMI. Add this AMI ID in the `consul_ami_id` variable in the `vars.tfvars` file.
```

Comment:
The reader will come across this instruction before they've read `terraform/README.md`, so it's not clear where `vars.tfvars` is supposed to be.